src/components/pages/Theme/Theme.jsx

```diff
@@ -1,886 +1,102 @@
-import React, { useState } from 'react';
-
-//redux datagrid
-import Datagrid, { PaginationHandler } from 'redux-form-datagrid';
+import React from 'react';
 
 // semantic
-import {
-  Button,
-  Checkbox,
-  Divider,
-  Dropdown,
-  Form,
-  Header,
-  Icon,
-  Input,
-  Label,
-  Message,
-  Modal,
-  Segment,
-  Select,
-  Tab,
-  Table,
-  TextArea,
-} from 'semantic-ui-react';
+import { Header, Segment } from 'semantic-ui-react';
+
+// segments
+import { ButtonSegment } from './Button/Button';
+import { CheckboxSegment } from './Checkbox/Checkbox';
+import { DatagridSegment } from './Datagrid/Datagrid';
+import { DividerSegment } from './Divider/Divider';
+import { DropdownSegment } from './Dropdown/Dropdown';
+import { FormLabelSegment } from './FormLabel/FormLabel';
+import { InputSegment } from './Input/Input';
+import { LabelSegment } from './Label/Label';
+import { MessageSegment } from './Message/Message';
+import { ModalSegment } from './Modal/Modal';
+import { TableSegment } from './Table/Table';
+import { TabSegment } from './Tab/Tab';
+import { TextAreaSegment } from './TextArea/TextArea';
+import { TypographySegment } from './Typography/Typography';
 
 // styles
 import styles from './Theme.module.css';
 
-const MessageExample = (props) => {
-  const [isMessageOpen, setIsMessageOpen] = useState(true);
-
-  const handleMessageDismissClick = () => {
-    setIsMessageOpen(false);
-  };
-
+const SectionWrapper = ({ title, children }) => {
   return (
-    <div>
-      {isMessageOpen && (
-        <Message onDismiss={handleMessageDismissClick} {...props} />
-      )}
+    <div className={styles.section}>
+      <Segment>
```

Comment:
I think the `<Segment>` and `<Header>` should live in each segment component.

This way, if we wanted to move them anywhere else, the segment and header would be included.

```
<div className={styles.section}>
  <ButtonSegment />
</div>
```

Reply:
Each segment is wrapped within `<Header>` and `<Segment>`. That's why I created a common component for this to reduce duplication:
```
const SectionWrapper = ({ title, children }) => {
  return (
    <div className={styles.section}>
      <Segment>
        <Header as='h2' dividing>
          {title}
        </Header>
        {children}
      </Segment>
    </div>
  );
};
```
Should I export it, so that child components can reuse it inside? I'll look something like this:
```
export const ButtonSegment = () => {
  return (
  <SectionWrapper title='Button'>
    <div className={styles.column}>
      <div className={styles.row}>
        <Button size='small'>Small Primary</Button>
        <Button size='medium'>Medium Primary</Button>
        <Button size='large'>Large Primary</Button>
      </div>
      <div className={styles.row}>
        <Button secondary size='small'>
          Small Secondary
        </Button>
        <Button secondary size='medium'>
          Medium Secondary
        </Button>
        <Button secondary size='large'>
          Large Secondary
        </Button>
      </div>
    </div>
  </ SectionWrapper>
  );
};
``` 
The code is exactly the same, that's why I want to minimize and reuse as much as possible.

Reply:
You _could_ export and reuse `<SectionWrapper>`, but I think it'll be a little more flexible if we add `<Segment>` and `<Header>` to each component, in case something needed to be adjusted for a particular segment down the road. I know it seems redundant, but I think that's ok in this case.

If you do decide to export and reuse it, I wouldn't include `<div className={styles.section}>`, as that is specific to the `<Theme>` layout.